share/smack/top.py

```diff
@@ -518,6 +518,12 @@ def arguments():
         default=False,
         help='enable support for string')
 
+    translate_group.add_argument(
+        '--fail-on-loop-exit',
```

Comment:
I think we may want a better name here. For example, CBMC's `--unwinding-assertions` makes more sense. Maybe we should call it `--enable-loop-unrolling-checks` ?